website/docs/docs/build/data-tests.md

```diff
@@ -70,6 +70,30 @@ The name of this test is the name of the file: `assert_total_payment_amount_is_p
 
 Singular data tests are easy to write—so easy that you may find yourself writing the same basic structure over and over, only changing the name of a column or model. By that point, the test isn't so singular! In that case, we recommend...
 
+<VersionBlock firstVersion="1.9">
+
+### Document singular tests
+
+You can set descriptions in your YAML files to document singular data tests:
+
+<File name='data_tests.yml'>
```

Comment:
While any of the [`docs-paths`](https://docs.getdbt.com/reference/project-configs/docs-paths) will work, the `tests` directory makes sense here because that is the default for [`test-paths`](https://docs.getdbt.com/reference/project-configs/test-paths). Furthermore, using `singular` not because it is required (it's not!), but because it is a very readable and logical complement to the `generic` subdirectory that contains definitions for custom generic data tests.

```suggestion
<File name='tests/singular/data_tests.yml'>
```